bindings/rust/s2n-tls/src/client_hello.rs

```diff
@@ -206,6 +132,256 @@ impl ClientHello {
     }
 }
 
+#[cfg(feature = "unstable-fingerprint")]
+pub use self::fingerprint::*;
+
+// Fingerprinting is an unstable feature. This module can be removed and added
+// to the client_hello module once we have settled on an implementation.
+#[cfg(feature = "unstable-fingerprint")]
+pub mod fingerprint {
+    use crate::error::{Error, Fallible};
+    use s2n_tls_sys::*;
+
+    use super::ClientHello;
+
+    #[derive(Copy, Clone)]
+    pub enum FingerprintType {
```

Comment:
Not in the scope of this PR, but we should probably mark this `non-exhaustive` before we do the actual `0.2` release.

Reply:
We already did the 0.2 release :) But we can mark it now, since fingerprinting is unstable anyway.